src/include/daos/btree.h

```diff
@@ -59,16 +59,31 @@ struct btr_node {
 	uint16_t			tn_flags;
 	/** number of keys stored in this node */
 	uint16_t			tn_keyn;
+	/** The last key in this node, used for lazy rebalance. */
+	uint16_t			tn_last_key;
```

Comment:
I guess this doesn't break durable format since it's a new feature using padding, right?

Reply:
The patch skip the old 32-bits pending into two fields, one is 16-bits tn_last_key, the other is 16-bits padding. So it is still compatible with old durable format.